project/src/main/packed-sprite.gd

```diff
@@ -79,6 +79,8 @@ Loads the Aseprite json file.
 Stores the offset and region_rect for each frame.
 """
 func _load_frame_data() -> void:
+	_frame_src_rects.clear()
+	_frame_dest_rects.clear()
```

Comment:
This is worth mentioning in the commit comment.

Reply:
Expanded commit comment, `Fixed bug where setting PackedSprite.frame_data to null did not empty the
cached sprite areas.`